components/navigation/navigation.css

```diff
@@ -0,0 +1,21 @@
+.navbar{
+  background-color:var(--preferred-color);
+  display:flex;
+  padding:0.7rem;
+  border-top-left-radius: 0.3rem;
+  border-top-right-radius: 0.3rem;
+  
+  
+}
+.nav-title{
+  padding-left:10px;
+  font-weight: 700;
+  color:var(--text-color);
+}
+.nav-pills{
+  display:flex;
+  flex-wrap:wrap;
+  color:var(--text-color);
+  margin-left:700px;
+  gap:20px;
+}
```

Comment:
While saving the css file, you can prettier to format the css file, you haven't used spaces after colon. For eg. gap:20px should be gap:<space>20px